src/mscorlib/src/System/Collections/Generic/KeyValuePair.cs

```diff
@@ -62,5 +62,12 @@ public struct KeyValuePair<TKey, TValue> {
             s.Append(']');
             return StringBuilderCache.GetStringAndRelease(s);
         }
+
+        //[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Hmm, the file is in CoreRT, but it is not actually building. And I see that this was discussed in the corefx issues. Still, I think it would be a fine idea to move it to CoreLib.